helm/ovn-kubernetes/values-single-node-zone.yaml

```
@@ -167,3 +167,8 @@ monitoring:
   # in defining that.
   commonServiceMonitorSelectorLabels:
     release: kube-prometheus-stack
+  # -- deploy ServiceMonitors for service discovery using the Prometheus Operator
+  enableServiceMonitor: true
+  # -- deploy PrometheusRules for specific metric collection using the Prometheus Operator
+  enablePrometheusRule: true
+
```

YAMLlint (1.37.1)

[warning] too many blank lines (1 > 0)

(empty-lines)